docs/user_guides/CASE_MANAGERS.md

```diff
@@ -0,0 +1,74 @@
+
+You can log a call to a patient by clicking the phone icon on the far right, which will open a pop-up with some additional information. Generally, you'll do this as follows:
+
+* Click the phone icon on the right side to pop up the additional information
+* Dial your phone to make the call
+* If the patient picks up and you start a conversation with them, click the `I reached the patient` button; this will redirect you to their Patient Record View page. Otherwise, if you leave a voicemail or it just rings, you can click the appropriate link to return to your call list.
+* Either action will shift them to the `Completed Calls` section.
+
+You can remove patients from your call list by clicking the red X icon. Your call list will automatically empty out after two weeks of not logging in.
+
+You can click the patient's name to go straight to their `Patient Record View`.
+
+### Urgent cases
+
+There is a way to flag patients so that they are easier for others to find. This section is where they show up. This is shared across everyone who logs in on the line.
+
+To mark a patient urgent, look in the `Patient Record View` under `Notes`.
+
+Patients automatically filter out of this section after a week of not being updated, or if they're marked resolved, or if a pledge is sent for them to a clinic.
+
+### Line Activity Log
+
+To make it easier to see who the previous case manager was communicating with, we have the `Line Activity Log`. DARIA keeps track of every call and pledge in the last three weeks, and displays them in this section for a given line.
+
+You can use this section to add particular patients to your call list, or click into their patient records.
+
+## The patient record view
+
+One of DARIA's major strengths is consistent organization of information. Our team has worked hard to present information about particular patients in ways that are easy to sort. This information is associated with a patient, and shows up the same for every case manager.
+
+Your fund will have guidance on what data to collect specifically.
+
+### Entering information about a patient
+
+Most fields on this page detect changes automatically, and autosave. (Logging notes, external pledges, and practical support requires pressing a button.)
+
+### Sending (or unsending) a pledge
+
+To send a pledge, click the button on the left sidebar. This will open a pop-up with information about the patient for you to confirm.
+
+If DARIA has all the information it needs, click next to go to the next screen. At a minimum, DARIA needs a clinic, appointment date, and amount of money to mark a pledge.
+
+DARIA will now prompt you to send the pledge to the clinic. Note that DARIA does not automatically send pledges to clinics. Ask your fund for guidance on how to send pledges.
+
+Finally, click next again, and check the box to indicate that you sent the pledge.
+
+To withdraw a sent pledge, click the `Cancel pledge` button, click next, and uncheck the box.
```

Comment:
Instead of saying "To withdraw a sent pledge," it might be more useful to say "If changes are needed, you can withdraw the pledge in Daria by clicking the 'cancel pledge' button, then next, and uncheck the box. Ask your fund for guidance on contacting the clinic to inform them of any changes to a pledge that was already sent."

Reply:
this is a great callout, thank you very much!